src/MSBuild/LiveLogger/LiveLogger.cs

```diff
@@ -20,14 +20,36 @@ internal class LiveLogger : ILogger
         public LoggerVerbosity Verbosity { get; set; }
         public string Parameters { get; set; }
 
+        /// <summary>
+        /// List of events the logger needs as parameters to the <see cref="ConfigurableForwardingLogger"/>.
+        /// </summary>
+        /// <remarks>
+        /// If LiveLogger runs as a distributed logger, MSBuild out-of-proc nodes might filter the events that will go to the main node using an instance of <see cref="ConfigurableForwardingLogger"/> with the following parameters.
+        /// </remarks>
+        public static readonly string[] ConfigurableForwardingLoggerParameters =
+        {
+            "BUILDSTARTEDEVENT",
+            "PROJECTSTARTEDEVENT",
+            "TARGETSTARTEDEVENT",
+            "TASKSTARTEDEVENT",
```

Comment:
Looks like we will need TASKFINISHED in current main:

https://github.com/dotnet/msbuild/blob/dfd8f413a80cd0865f968b2c0ad9b09c0df8c430/src/MSBuild/LiveLogger/LiveLogger.cs#L219-L232

Reply:
No, TASKFINISHED is currently commented out. This is dead code.